app/presenters/hyku/work_show_presenter_decorator.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
is there a good reason not to add these to Hyku directly?

Reply:
I think the `Hyku::WorkShowPresenter` could be a lot smarter; namely delegate all method missing to the underlying `SolrDocument`.  But yes, this should be pushed down into Hyku.

Reply:
remove this after double_combo update